### PR TITLE
Document precisely what keys and certificates are enrolled

### PR DESCRIPTION
**Pull request by kashyapc**
Thanks-to: Laszlo Ersek <lersek@redhat.com>
Signed-off-by: Kashyap Chamarthy <kchamart@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,3 +42,27 @@ looks as follows:
...
+
+The following certificates and keys are enrolled by the tool:
+
+  - As *Platform Key*, and as one of the two *Key Exchange Keys* that we
+    set up, the `EnrollDefaultKeys.efi` binary on both Fedora and RHEL,
+    uses the same the digital certificate called `Red Hat Secure Boot
+    (PK/KEK key 1)/emailAddress=secalert@redhat.com`, and the Red Hat's
+    Product Security team has the private key for it.
+
+  - The certificate that is enrolled as the second *Key Exchange Key& is
```
Comment (kashyapc) [inline README.md]: Darn, spurious "&". It should be a "*"

File: README.md [hunk 2]
```diff
@@ -40,3 +42,27 @@ looks as follows:
...
+
+
+## What certificates and keys are enrolled?
+
+The following certificates and keys are enrolled by the tool:
+
+  - As *Platform Key*, and as one of the two *Key Exchange Keys* that we
+    set up, the `EnrollDefaultKeys.efi` binary on both Fedora and RHEL,
+    uses the same the digital certificate called `Red Hat Secure Boot
+    (PK/KEK key 1)/emailAddress=secalert@redhat.com`, and the Red Hat's
```
Comment (puiterwijk) [inline README.md]: s/the//

File: README.md [hunk 3]
```diff
@@ -40,3 +42,27 @@ looks as follows:
...
           [    3.261277] EFI: Loaded cert 'Fedora Secure Boot CA: fde32599c2d61db1bf5807335d7b20e4cd963b42' linked to '.builtin_trusted_keys'
+
+
+## What certificates and keys are enrolled?
+
+The following certificates and keys are enrolled by the tool:
+
+  - As *Platform Key*, and as one of the two *Key Exchange Keys* that we
+    set up, the `EnrollDefaultKeys.efi` binary on both Fedora and RHEL,
+    uses the same the digital certificate called `Red Hat Secure Boot
```
Comment (puiterwijk) [inline README.md]: s/same the/same/

**Review by puiterwijk — COMMENTED** [review]
Two typos pointed out by Laszlo.

**Review by puiterwijk — DISMISSED** [review]

**Pull request merged**